apprentice_teacher.rb

```diff
@@ -1,38 +1,18 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
-
+require_relative 'instructor'
+class ApprenticeTeacher < Instructor
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 800
-    @phase = 3
-  end
-
-  def offer_high_five
-    "High five!"
-  end
-
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
+    super
+    @target_raise = 800# possibly make this  a constant
```

Comment:
+1 making it a constant.  What differences are there between constants and instance variables?